src/components/body.js

```diff
@@ -79,14 +79,14 @@ export default class Body extends Component {
   constructor(props, context) {
     super(props);
 
-    const { args, children, shape, ...options } = props;
```

Comment:
The weird thing with object spread is sometimes, folks use this to get **rid** of `children` for the resulting `options` variable. We should check that it is OK for `options.children` to now be added.

Reply:
So it looks like these options are passed into a constructor in `matter.js`. It looks like it's fine if `children` is left in, but I think it's better for maintenance to keep it clean and not pass in superfluous arguments. I'll revert these object spread and add an `eslint-disable`.